service/src/main/java/skills/services/admin/SkillsAdminService.groovy

```diff
@@ -231,6 +234,11 @@ class SkillsAdminService {
                     pointIncrementDelta = 0
                 }
                 totalPointsRequested = skillRequest.pointIncrement * skillRequest.numPerformToCompletion
+
+                def finalizeState = skillCatalogFinalizationService.getCurrentState(skillRequest.projectId)
+                if (finalizeState != SkillCatalogFinalizationService.FinalizeState.COMPLETED) {
+                    skillRequest.enabled = false
```

Comment:
I think the logic needs to be ``skillRequest.enabled = isCurrentlyEnabled`` because it's possible to have multiple import/finalize runs; if this skill was already finalized then it would get reset mistakenly to the disabled state; alternatively the `if` statement can be enhanced to ``if (finalizeState != SkillCatalogFinalizationService.FinalizeState.COMPLETED && !isCurrentlyEnabled)``; please write another service tests to facilitate this scenario and make sure it doesn't mistakenly disable previously imported and finalized skills